server/router/driver.ts

```diff
@@ -12,6 +12,50 @@ import { UserType } from '../models/subscription';
 const router = express.Router();
 const tableName = 'Drivers';
 
+// Get all drivers
+router.get('/', validateUser('Admin'), (req, res) => {
+  db.getAll(res, Driver, tableName);
+});
+
+// Get all available drivers at a specific date and time
+router.get('/available', validateUser('User'), (req, res) => {
+  const { date, startTime: reqStartTime, endTime: reqEndTime } = req.query;
+  const numToDay = {
+    1: 'Mon',
+    2: 'Tue',
+    3: 'Web',
+    4: 'Thu',
+    5: 'Fri',
+  };
```

Comment:
nice change! maybe use an array here instead? theres a little too much typecasting going on for `reqDate`. you can put placeholders (like undefined) for array position 0 and 7

Reply:
Done